### DISCUSSION TITLE
Thoughts on an Astro Sphinx Theme for documentation

### DISCUSSION THREAD

**Discussion by tdinino** [RocketCommunicationsInc/astro#1273] [Q&A]
The Astro standard has done a great job of standardizing the look and feel of DoD related products.  With Sphinx being one of the larger documentation building tools, has the team thought about extending this standard to documentation as well with a specific Astro Sphinx Theme?  e.g. sphinx_rtd_theme, Alabaster, etc.

**Comment by dmcalester** [comment]
Hey there. We have not as yet considered Sphinx, it’s actually the first time being made aware of the product, but it looks very interesting. Do you know if Sphinx has been FedRamp approved and/or do you know of many teams/programs/organizations using it?

**Reply by tdinino** [reply]
We have looked in the past and believe that Sphinx is FedRamp approved.  I do not have a link I can share, however.  As for projects that are using or have used Sphinx, the Space Rapid Capabilities Office had/has a program and the Forge Program.  I do not know if there are any other teams using Sphinx.
